### PR TITLE
Feat/eventdb crud

### PR DESCRIPTION
**Pull request by binayak9932**
issue #242 
continuation of  pr  #246  #251 #252
- [X] Create new EventBaseModel(Base)[here](https://github.com/CarmineOptions/derisk-research/blob/master/data_handler/db/models/event.py), [Base](https://github.com/CarmineOptions/derisk-research/blob/master/data_handler/db/models/base.py#L11)
event_name: str, index=True
block_number: int index=True
protocol_id: str, index=True [ProtocolIDs choices](https://github.com/CarmineOptions/derisk-research/blob/master/shared/constants.py#L55)
- [X] Create in [zklend_events.py](https://github.com/CarmineOptions/derisk-research/blob/master/data_handler/db/models/zklend_events.py) new models based on pydantic fields and inherited from EventBaseModel
- [X] Create CRUD class [here](https://github.com/CarmineOptions/derisk-research/blob/master/data_handler/db/crud.py) new class
ZkLendEventDBConnecotr(DBConnector) with next methods:
create_accumulator_event(protocol_id, event_name, block_number, event_data)
create_liquidation_event(protocol_id, event_name, block_number, event_data)
get_all_events(protocol_id, event_name, block_number)
- [X] Generate migration file

new pr cause of merge conflict

### CODE REVIEW

File: apps/data_handler/db/models/zklend_events.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    """
+    Database model for AccumulatorsSync event, inheriting from EventBaseModel.
+
+    This model stores the token address and the converted values for lending and debt
+    accumulators as Decimal.
+    """
+
+    __tablename__ = "accumulators_sync_event_data"
+
+    id: Mapped[UUID] = mapped_column(UUID, primary_key=True, default=uuid4)
```
Comment (djeck1432) [inline apps/data_handler/db/models/zklend_events.py:20]: EventBaseModel model has already `id` field, please, remove it

File: apps/data_handler/db/models/zklend_events.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+class LiquidationEventData(EventBaseModel):
+    """
+    Database model for Liquidation event, inheriting from EventBaseModel.
+
+    This model stores the details of a liquidation event, including addresses and amounts.
+    """
+
+    __tablename__ = "liquidation_event_data"
+
+    id: Mapped[UUID] = mapped_column(UUID, primary_key=True, default=uuid4)
```
Comment (djeck1432) [inline apps/data_handler/db/models/zklend_events.py:39]: EventBaseModel model has already id field, please, remove it

**Review by djeck1432 — CHANGES_REQUESTED** [review]
Small changes, all others looks good to me

**Comment by binayak9932** [comment]
@djeck1432  review pls

**Review by djeck1432 — APPROVED** [review]
Looks good to me

**Pull request merged**